msgraph-connector-test/pom.xml

```diff
@@ -69,10 +69,10 @@
       <plugin>
         <groupId>com.axonivy.ivy.ci</groupId>
         <artifactId>project-build-plugin</artifactId>
-        <version>9.3.2</version>
+        <version>9.4.1</version>
         <extensions>true</extensions>
         <configuration>
-          <ivyVersion>9.4.0</ivyVersion>
+          <ivyVersion>[9.4.0,]</ivyVersion>
```

Comment:
so far so good; maybe it would have been better to omit the 'ivyVersion' configuration all-together ... so that migrations to newer plugin versions will use the default configured ivy-version

Reply:
yes for sure. I think we'll streamline this connector when we have 9.4 released. Because this is the only one connector which is built on 9.4.0 (which is not released yet)

Later we should handle all connectors the same way.